website/docs/r/global_cluster_config.html.markdown

```diff
@@ -13,6 +13,7 @@ description: |-
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
 
+~> **IMPORTANT:** A Global Cluster Configuration, once created, can only be deleted and never re-created again with the same data if not without the help of the Atlas UI. This is because the configuration and its related collection with shard key and indexes are managed separately and they would end up in an inconsistent state. [Read more about Global Cluster Configuration](https://www.mongodb.com/docs/atlas/global-clusters/)
```

Comment:
Should we include about this in the release guide too? 
Not sure if this should be considered a breaking change, I understand we are removing an incorrect behavior (just thinking out loud)

Reply:
It should not be considered a breaking change because it's a bug. I wasn't planning to put it in the release guide, but let me know otherwise